Makefile

```diff
@@ -26,6 +26,10 @@ bootstrap:
 test-integration:
 	go test ./tests/... ${TEST_OPTS}
 
+.PHONY: test-integration
+test-integration-parallel:
+	ginkgo -slowSpecThreshold=120.00 --nodes=30 tests/
```

Comment:
`ginkgo --nodes=30` looks equivalent to `go test -ginkgo.parallel.total=30`. If so, this could use `${TEST_OPTS}` as other Makefile targets do.

Also, why does this target test `tests/` but the one above it tests `./tests/...`? In this case they amount to the same thing, but they should be consistent.